### PR TITLE
Add runbook

### PR DESCRIPTION
**Pull request by bboykov**
# Description

## What

Add runbook for the service.

## Why

[JIRA ticket](https://financialtimes.atlassian.net/browse/UPPSF-1181).

## Scope and particulars of this PR (Please tick all that apply)

- [ ] Tech hygiene (dependency updating & other tech debt)
- [ ] Bug fix
- [ ] Feature
- [X] Documentation
- [ ] Breaking change
- [X] Minor change (e.g. fixing a typo, adding config)

___
This Pull Request follows the rules described in our [Pull Requests Guide](https://github.com/Financial-Times/upp-docs/tree/master/guides/pr-guide)

### CODE REVIEW

File: runbooks/runbook.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
+# UPP - Public Things API
+
+Provides a publicly available API that gives basic information about things in the graph store. It is written in Go, uses Neo4j as a data store, and is deployed in the UPP Kubernetes stack
```
Comment (karsov) [inline runbooks/runbook.md:3]: I would leave only the first sentence here and have a corrected description in the "Architecture" section.

File: runbooks/runbook.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+- marina.chompalova
+- miroslav.gatsanoga
+
+## Host Platform
+
+AWS
+
+## Architecture
+
+Provides a publicly available API that gives basic information about things in the graph store. It is written in Go, uses Neo4j as a data store, and is deployed in the UPP Kubernetes stack
```
Comment (karsov) [inline runbooks/runbook.md:43]: This is not correct for quite some time. My suggestion:
```suggestion
The service calls Public Concepts API to retrieve the information from the Neo4j data store.
It is deployed in the UPP Delivery Kubernetes clusters.
```

File: runbooks/runbook.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+
+## Code
+
+public-things-api
+
+## Primary URL
+
+<https://github.com/Financial-Times/public-things-api>
```
Comment (karsov) [inline runbooks/runbook.md:11]: This is a public API, its URL is:
```suggestion
<http://api.ft.com/things>
```

File: runbooks/runbook.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+
+FullyAutomated
+
+## Failback Process Type
+
+PartiallyAutomated
+
+## Failover Details
+
+See the [failover guides for more details](https://github.com/Financial-Times/upp-docs/tree/master/failover-guides)
```
Comment (karsov) [inline runbooks/runbook.md:67]: I would prefer to have here the cluster specific failover guide:
```suggestion
The service is deployed in both Delivery clusters. The failover guide for the cluster is located here: https://github.com/Financial-Times/upp-docs/tree/master/failover-guides/delivery-cluster
```

File: runbooks/runbook.md [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+## Key Management Details
+
+There is no key rotation procedure for this system.
+
+## Monitoring
+
+Look for the pods in the cluster health endpoint and click to see pod health and checks:
+
+- <https://upp-prod-delivery-eu.upp.ft.com/__health>
+- <https://upp-prod-delivery-us.upp.ft.com/__health>
```
Comment (karsov) [inline runbooks/runbook.md:102]: The specific links would be way more useful:
```suggestion
- Delivery-Prod-EU health: <https://upp-prod-delivery-eu.upp.ft.com/__health/__pods-health?service-name=public-things-api>
- Delivery-Prod-US health: <https://upp-prod-delivery-us.upp.ft.com/__health/__pods-health?service-name=public-things-api>
```

**Review by karsov — CHANGES_REQUESTED** [review]
The dependency on `public-concepts-api` is missing.

File: runbooks/runbook.md [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+It is deployed in the UPP Delivery Kubernetes clusters.
+
+## Contains Personal Data
+
+No
+
+## Contains Sensitive Data
+
+No
+
```
Comment (karsov) [inline runbooks/runbook.md:53]:
```suggestion

## Dependencies

- public-concepts-api

```

**Review by karsov — APPROVED** [review]

**Review by ivan-p-nikolov — APPROVED** [review]

**Pull request merged**